src/DatadogLoggingService.js

```diff
@@ -60,6 +60,8 @@ class DatadogLoggingService extends NewRelicLoggingService {
       sessionSampleRate: parseInt(process.env.DATADOG_SESSION_SAMPLE_RATE || 0, 10),
       sessionReplaySampleRate: parseInt(process.env.DATADOG_SESSION_REPLAY_SAMPLE_RATE || 0, 10),
       trackUserInteractions: true,
+      trackSessionAcrossSubdomains: true,
```

Comment:
Based on the documentation:

> Options that must have matching configuration when you are using the Logs Browser SDK:

> Options that must have a matching configuration when using the RUM SDK:

Given this, I wonder if it might make sense to abstract common options across `datadogRum.init` and `datadogLogs.init` so it doesn't necessitate manually supplying the same values in both; might help to guarantee they are always matching, for example. Something like:

```jsx
const datadogVersion = process.env.DATADOG_VERSION || process.env.APP_VERSION || '1.0.0';
const commonInitOptions = {
  clientToken: process.env.DATADOG_CLIENT_TOKEN,
  site: process.env.DATADOG_SITE || '',
  env: process.env.DATADOG_ENV || '',
  service: process.env.DATADOG_SERVICE || '',
  version: datadogVersion,
  trackSessionAcrossSubdomains: true,
  usePartitionedCrossSiteSessionCookie: true,
};
datadogRum.init({
  ...commonInitOptions,
  beforeSend: this.beforeSend,
  sessionSampleRate: parseInt(process.env.DATADOG_SESSION_SAMPLE_RATE || 0, 10),
  sessionReplaySampleRate: parseInt(process.env.DATADOG_SESSION_REPLAY_SAMPLE_RATE || 0, 10),
  trackUserInteractions: true,
  trackResources: true,
  trackLongTasks: true,
  defaultPrivacyLevel: process.env.DATADOG_PRIVACY_LEVEL || 'mask-user-input',
});
datadogLogs.init({
  ...commonInitOptions,
  forwardErrorsToLogs: true,
  sessionSampleRate: parseInt(process.env.DATADOG_LOGS_SESSION_SAMPLE_RATE || 0, 10),
});
```

Reply:
I like this, implementing.